package Exception;

public class square {

	public static void main(String[] args) {
		// TODO Auto-generated method stub

		try 
		{
			Demo.Square();
		}
		catch (ArithmeticException e) 
		{
			e.printStackTrace();
		}
	}
}
